editor/src/clj/editor/editor_extensions/commands.clj

```diff
@@ -135,7 +135,7 @@
                              "Bundle" :global
                              "Debug" :global
                              "Edit" :global
-                             "Outline" :outline
+                             "Outline" :global
```

Comment:
I had to change the context to get the selected nodes from the `env`, but I am not sure why we need to filter by context here (meaning that `:global` would work for all cases here).

Reply:
Hmm, I think the `:outline` context might be required here so that we get selection from a selected tree item in the outline, and not from, e.g., scene.

Reply:
isn't the selection context the same for both? So, if you select something on the outline and then right click on the scene, the selection will be altered, because they share the same selection provider. So if you cannot click on it on the scene, it will be deselected, unless you deliberately hold shift?